src/notifier/__init__.py

```diff
@@ -60,17 +63,17 @@ class Notifier(ABC):
     Pushover, E-mail, Slack, WhatsApp, etc
     """
 
-    def __init__(self, title_prefix: str, config: dict):
+    def __init__(self, title_prefix: str, config: ConfigView):
         self._title_prefix = title_prefix
         self._config = config
         self._conn_timeout_seconds = 10
         self._notification_types = [EventType.USER]
         self._notification_services = [EventService.HARVESTER, EventService.FARMER, EventService.FULL_NODE]
 
-        daily_stats = config.get("daily_stats", False)
-        wallet_events = config.get("wallet_events", False)
```

Comment:
Can we once again go through the defaults for all values and make sure we’re keeping them the same please?

Here `daily_stats` used to have been disabled by default. But the new `default_config.yaml` has it enabled. Same thing applies for `wallet_events`.

I’d like to keep the migration to `confuse` separate of any decisions on changing the existing defaults as this will require more nuanced discussions on backward compatability and will delay this PR. Concrete example: there are users with multiple notifiers with legacy configs that don’t override the `wallet_event` values. They might start getting overwhelming number of notifications from pool rewards if this gets merged.